dozer-tests/src/e2e_tests/checker/mod.rs

```diff
@@ -56,8 +56,10 @@ fn assert_command_fails(mut command: Command, expected_string_in_stdout: Option<
     if output.status.success() {
         panic!("Command {command:?} is expected to fail, but it succeeded");
     }
-    let stdout = String::from_utf8_lossy(&output.stdout);
-    error!("{}", stdout);
+    let stdout = String::from_utf8(output.stdout).unwrap();
```

Comment:
This is my debug code. We should remove it.

Reply:
removed